client/my-sites/sidebar-unified/add-new-site.jsx

```diff
@@ -1,20 +1,22 @@
 import PropTypes from 'prop-types';
 import { Fragment } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import useOnboardingUrl from 'calypso/components/site-selector/use-onboarding-url';
 import TranslatableString from 'calypso/components/translatable/proptype';
 import SidebarCustomIcon from 'calypso/layout/sidebar/custom-icon';
 import SidebarItem from 'calypso/layout/sidebar/item';
 import SidebarSeparator from 'calypso/layout/sidebar/separator';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { getCurrentUser } from 'calypso/state/current-user/selectors';
-import getOnboardingUrl from 'calypso/state/selectors/get-onboarding-url';
 import { setLayoutFocus } from 'calypso/state/ui/layout-focus/actions';
 
 export const AddNewSite = ( { title, icon } ) => {
 	const reduxDispatch = useDispatch();
+	const onboardingUrl = useOnboardingUrl();
 	const visibleSiteCount = useSelector( getCurrentUser ).visible_site_count;
-	const onboardingUrl = useSelector( getOnboardingUrl );
-
+	if ( ! onboardingUrl ) {
+		return null;
```

Comment:
I couldn't use the "busy" property here, so I think the best is just to delay rendering until onboardingUrl becomes available.